charts/stable/mealie/templates/common.yaml

```diff
@@ -14,6 +14,10 @@ additionalContainers:
     env:
       - name: API_PORT
         value: "9000"
+      {{- range $name, $value := .Values.api.env }}
+      - name: {{ $name }}
+        value: {{ $value }}
+      {{- end }}
```

Comment:
Does not work when passing down bool values as string as the quotes are removed.
See [here](https://discord.com/channels/673534664354430999/740734177279868949/982910175788826724)
e.g. with:
```yaml
ALLOW_SIGNUP: "false"
```
This throws:
> json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string

Reply:
Ah, should probably be `$value | quote` then. I can make that change later today potentially, but for now you can add in extra quotes to your values. E.g. `myValue: '"false"'`

Reply:
Resolve by #1606